beat/heart.py

```diff
@@ -3786,8 +3754,14 @@ def init_wavemap(
     wmap : :class:`WaveformMapping`
     """
     wc = waveformfit_config
+    
+    if wc.spectrum_include:
```

Comment:
remove wc.spectrum_include and put right away domain there

Reply:
it's not easy to get rid of this option! we need to put domain in config file as well!

Reply:
that was what my comment was supposed to mean